README.rst

```diff
@@ -73,23 +73,33 @@ Clone the Repository
 
 To clone this repository you may use the following: ::
 
-    git clone git@github.com:xmos/lib_xcore_math 
-
+    git clone git@github.com:xmos/lib_xcore_math
 
 Including lib_xcore_math in External Applications
 -------------------------------------------------
 
+``lib_xcore_math`` can be compiled for both x86 platforms and xs3a architecture.
```

Comment:
Please change `and xs3a architecture` to either `and the xs3a architecture` or `and xs3a processors`.

Reply:
I used the second option.